dishes/chicken-biryani.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Chicken Biryani</title>
+</head>
+<body>
+    <h1>Chicken Biryani</h1>
+    <img src="https://www.licious.in/blog/wp-content/uploads/2022/06/chicken-hyderabadi-biryani-01.jpg" height="auto"
+        width="250px" alt="image for Chicken Biryani">
```

Comment:
`alt` attributes are not supposed to contain words like `image of` or `photo of`. This attribute is associated with accessibility & screen-readers usually announce the prefix keywords with the actual value of the `alt`

Update everywhere pls.